floop-data-export.py

```diff
@@ -48,7 +50,8 @@
 # Convert back to list so json.dump can process it
     arr = set(cList)
     new_arr = list(arr)
-
+# Create new variable to store first "N" conversations
+    new_arr_n = new_arr[:n]
```

Comment:
We don't want to limit the number of entries this far down in the process--we should implement the number of conversations above on line 28. Otherwise, we are pulling 10000 Conversations first, then limiting it to N from that.  The script will run faster if we only pull N to begin with

Reply:
Okay, Mike. I have hard coded n as 100 for now, and we can integrate to CLI in coming sprints